packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -88,7 +113,17 @@ const metadata = {
 		 * @slot sortItems
 		 * @public
 		 */
-		 "sortItems": {
+		 sortItems: {
+			type: HTMLElement,
+		},
+
+		/**
+		 * Defines the <code>filterItems</code> list.
+		 * @type {sap.ui.webcomponents.fiori.ListItem}
```

Comment:
Here we say use @type {sap.ui.webcomponents.fiori.ListItem}, but it's not entirely right.
The type is different now - FilterItem for filterItems and SortItem for sortItems. Also, I recommend adding a jsdoc to let the users know they need to import the SortItem, FIlterItem, FilterItemOption, because they are not auto-imported. Could be a note in the slots description or in the overview, or both.